common/base/src/base/string_func.rs

```diff
@@ -115,22 +117,30 @@ pub fn replace_nth_char(s: &str, idx: usize, newchar: char) -> String {
         .collect()
 }
 
-/// Return prefix of string.
+/// Return prefix of string(only support ASCII char).
 /// "a" -> "b"
 /// "1" -> "2"
 /// [96,97,127] -> [96,98,127]
 /// [127] -> [127, 127]
 /// [127,127,127, 127] -> [127,127,127, 127, 127]
-pub fn prefix_of_string(s: &str) -> String {
+pub fn prefix_of_string(s: &str) -> common_exception::Result<String> {
```

Comment:
The naming is very confusing to me. Can we come up with a better name here?

I can't recognize the input and output with `prefix_of_string`.

Reply:
ok, seems that `prefix_of_string` is not suitable in `string_func.rs`, i move it to `kv_api`.